timeseries/src/autogluon/timeseries/trainer/abstract_trainer.py

```diff
@@ -34,16 +35,15 @@
 logger = logging.getLogger("autogluon.timeseries.trainer")
 
 
-# TODO: This class is meant to be moved to `core`, where it will likely
-# TODO: be renamed `AbstractTrainer` and the current `AbstractTrainer`
-# TODO: will inherit from this class.
-# TODO: add documentation for abstract methods
-class SimpleAbstractTrainer:
-    trainer_file_name = "trainer.pkl"
-    trainer_info_name = "info.pkl"
-    trainer_info_json_name = "info.json"
-
+# TODO: This class will be removed after the behavior is reconciled with core
+class SimpleAbstractTrainer(AbstractTrainer):
     def __init__(self, path: str, low_memory: bool, save_data: bool, *args, **kwargs):
+        super().__init__(
+            path=path,
+            low_memory=low_memory,
+            save_data=save_data,
+        )
+
```

Comment:
For the init code below this, you should be able to remove pretty much all of it right? Looks like code-dupe atm.

Reply:
Yes. I will also get rid of this class altogether shortly.